tests/api.c

```diff
@@ -10362,7 +10362,7 @@ static void msg_cb(int write_p, int version, int content_type,
                    const void *buf, size_t len, SSL *ssl, void *arg)
 {
     (void)write_p;
-		(void)version;
+    (void)version;
 		(void)content_type;
```

Comment:
Looks like there are a few more hard tabs.  There are a few helpful ways that I have found to identify hard tabs:

1)  In the GitHub web view of a PR, view the "Files changed".  This allows you to see spacing differences.

2)  In vim, you can search for all hard tabs using:

/\t

This is referenced here:
http://vim.wikia.com/wiki/Highlight_unwanted_spaces